etcdserver/api/v3rpc/maintenance.go

```diff
@@ -128,11 +128,11 @@ func (ms *maintenanceServer) Snapshot(sr *pb.SnapshotRequest, srv pb.Maintenance
 }
 
 func (ms *maintenanceServer) Hash(ctx context.Context, r *pb.HashRequest) (*pb.HashResponse, error) {
-	h, rev, err := ms.kg.KV().Hash()
```

Comment:
still need to call `Hash()` if `r.Revision == 0`, otherwise the functional tester won't check all the buckets